lib/apis/moray.js

```diff
@@ -586,8 +596,25 @@ Moray.prototype.putVm = function (uuid, vm, cb) {
     this.connection.putObject(VMS_BUCKET_NAME, uuid, object, cb);
 };
 
+/*
+ * Deletes *all* VMs that match the filter generated from "params".
+ */
```

Comment:
I'm sorry, I don't think I was especially clear.  I meant "we don't allow" in the general VMAPI model.  I couldn't see any existing API in this Moray abstraction that will call `delObject()` or `deleteMany()` on the `vmapi_vms` bucket.

I think it's fine to leave it in there, but you should note that the general VMAPI data model doesn't involve actually deleting records from `vmapi_vms`, and that this is just internal API for testing/etc.